docs/configuration/config-file-reference.md

```diff
@@ -2198,6 +2198,17 @@ bucket_store:
   # CLI flag: -experimental.tsdb.bucket-store.meta-sync-concurrency
   [meta_sync_concurrency: <int> | default = 20]
 
+  # Whether the bucket store should use the binary index header. If false, it
+  # uses the legacy JSON index header.
```

Comment:
Calling old one "legacy" may be little premature, given that binary index header is still experimental.

Reply:
Fair enough. I've been too much optimistic. Fixed!